fabfile/config.py

```diff
@@ -12,6 +12,13 @@
 HOME_DIRECTORY = HOME_DIR
 RESULTS_DIRECTORY = os.path.join(HOME_DIR, 'results')
 CITUS_INSTALLATION = os.path.join(HOME_DIR, 'citus-installation')
+RELATIVE_REGRESS_PATH = 'src/test/regress'
+COMMUNITY_REPO = 'community'
+ENTERPRISE_REPO = 'enterprise'
```

Comment:
This does not contain the enterprise repo and there is already a variable called `ENTERPRISE_REPO`, which contains the right path for enterprise repo so we should remove this variable too.

Reply:
Done